src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -75,6 +70,16 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
     }
 
     /**
+     * Return the names of the classes mapped in this file.
+     *
+     * @return string[] The classes names
+     */
+    public function getMappedClasses()
```

Comment:
I wasn't sure about this. I thought it may be a bit overkill just for a single method.